go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/redhat-cop/namespace-configuration-operator
 
 require (
+	github.com/Masterminds/semver v1.5.0 // indirect
+	github.com/Masterminds/sprig v2.22.0+incompatible // indirect
 	github.com/openshift/api v0.0.0-20200205133042-34f0ec8dab87
 	github.com/operator-framework/operator-sdk v0.18.1
-	github.com/pkg/errors v0.9.1
 	github.com/redhat-cop/operator-utils v0.3.4
```

Comment:
I release a new version of operator utils 0.3.5, please pin that.
Also how did it even compile without the right version?

Reply:
I copied the operator util folder from my go/src folder to the mod folder for that specific version and then did a go mod vendor for the namespace config operator.

Reply:
ok, this was a good approach for local deevlopment, but it will not work for others. we need to update the go.mod to import the right version.

Reply:
I will look out for that in the future, thanks!

Reply:
Updated